msteams-platform/sbs-inmeeting-document-signing.yml

```diff
@@ -0,0 +1,473 @@
+
+    ### Check for sideloading option
+
+       After creating the app, you must load your app in Teams without distributing it. It's known as sideloading. Sign in to your Microsoft 365 account to view this option.
+
+       > [!NOTE]
+       > Sideloading is necessary for previewing and testing apps in Teams local environment. Enable Sideloading to preview and test your app in Teams locally.
+
+       Do you already have a tenant, and do you have the admin access? Let's check if you really do!
+
+    To verify sideload apps in Teams:
+
+    1. In the Teams client, select **Store**.
+    1. Select **Manage your apps**.
+    1. Select **Publish an app**.
+    1. Look for the option to **Upload a custom app**. If you see the option, sideloading apps is enabled.
+
+        :::image type="content" source="~/assets/images/bots/upload-custom-app1.png" alt-text="Illustration shows the option to upload a custom app in Teams.":::
+
+        > [!NOTE]
+        > Contact Teams administrator, if you don't find the option to upload a custom app.  
+
+    ## Download the code sample
+
+      You must use the [In-Meeting Document Signing](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-share-to-stage-signing/csharp) code sample for this step-by-step guide. To use the code sample, clone the repository from GitHub.
+
+- title: Create and register your app
+  durationInMinutes: 5
+  content: |
+    The following steps help you to create and register your app in Azure portal.
+
+    * Create an app registration in Azure AD.
+    * Use ngrok to create a tunnel.
+    
+    ## Create an app registration in Azure AD
+
+    Register a new app in Azure AD, and configure the tenancy and app's platform. You'll generate a new app ID that will be updated later in your Teams app manifest file.
+
+    ### To register a new app in Azure AD
+
+    1. Open the [Azure portal](https://ms.portal.azure.com/) on your web browser.
+    The Microsoft Azure AD Portal page opens.
+
+    2. Select the **App registrations** icon.
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/azure-portal.png" alt-text="Azure AD Portal page.":::
+
+        The **App registrations** page appears.
+
+    3. Select **+ New registration** icon.
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/app-registrations.png" alt-text="New registration page on Azure AD Portal.":::
+
+        The **Register an application** page appears.
+
+    4. Enter the name of your app that you want to be displayed to the app user. You can change this name at a later stage, if you want to.
+
+        :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/register-app.png" alt-text="App registration page on Azure AD Portal.":::
+
+    5. Select **Multitenant* as the type of user account that can access your app.
```

Comment:
your styling is wrong here. You've two \* at the start and only one at the end. so currently it's rendering as "**Multitenant*"